go.mod

```diff
@@ -36,14 +36,16 @@ require (
 
 replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.2+incompatible // Required by OLM
-	//github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.5.1
 	github.com/operator-framework/operator-sdk => github.com/operator-framework/operator-sdk v1.0.1
 
+	// Scanners flag anything less than v0.3.6 as having a CVE.
+	golang.org/x/text => golang.org/x/text v0.3.7
+
 	k8s.io/api => k8s.io/api v0.21.7
 	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.21.7
 	k8s.io/apimachinery => k8s.io/apimachinery v0.21.7
 	k8s.io/apiserver => k8s.io/apiserver v0.21.7
-	k8s.io/client-go => k8s.io/client-go v0.21.7 // Required by prometheus-operator
```

Comment:
Huh, my assumption was that the comment was wrong and/or out of date, and that it didn't provide any value because it doesn't actually explain _why_ prometheus operator needs that pin.